scripts/deploy.sh

```diff
@@ -13,7 +13,7 @@ else
   git push --force heroku $CIRCLE_SHA1:refs/heads/master
 fi
 
-if [ "$DEPLOY_ENV" = 'production' ]
+if [ "$DEPLOY_ENV" = 'production' ]; then
```

Comment:
Lacking a `then` here caused a syntax error but the [build](https://circleci.com/gh/artsy/force/6829) was still 💚 .

![screen shot 2018-06-21 at 1 54 36 am](https://user-images.githubusercontent.com/796573/41700425-3d9fba40-74f6-11e8-80bc-91fb1508096b.png)

Reply:
Nice catch